### PR TITLE
Provide status responses reference from BloomNetClient

### PR DESCRIPTION
**Pull request by ace-subido**
- Add STATUSES constant at the `Txn` object for status reference

### CODE REVIEW

File: lib/bloom_net_center_client/models/txn.rb [hunk 1]
```diff
@@ -22,5 +22,12 @@ class Txn
 
     ATTRS = self.attribute_set.map(&:name)
 
+    STATUSES = {
+      unfunded: "unfunded",
+      funded: "funded",
+      claiming: "claiming",
+      claimed: "claimed",
```
Comment (ramontayag) [inline lib/bloom_net_center_client/models/txn.rb:29]: What's the reason behind making the keys and values separate entries if they're the same text?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/) 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
```
Comment (ramontayag) [inline CHANGELOG.md:4]: 👍

File: lib/bloom_net_center_client/models/txn.rb (hunk 1 above)
Comment (ace-subido) [reply lib/bloom_net_center_client/models/txn.rb:29]: oh, I was thinking we could use this as: `if remote_txn.status == BloomNetCenterClient::Txn::STATUSES[:funded]`  operations. Is there a better way, or a more concise way for implementing some sort of status helper?

**Comment by ramontayag** [comment]
Ah no that's ok! Just wondering.

**Review by ramontayag — APPROVED** [review]

**Comment by ace-subido** [comment]
ah, ok ok 😄

**Pull request merged**